Upgrade notion dependency, add code, quote and image (#5)
instantish/martian: README.md

uses [unified](https://github.com/unifiedjs/unified) to create a Markdown AST, then converts the AST into Notion
objects.

> Designed to make using the Notion SDK and API easier.  Notion API version 0.4.4.

Reviewer: ```suggestion
Designed to make using the Notion SDK and API easier.  Notion API version 0.4.5.
```

### Supported Markdown Elements

* All inline elements (italics, bold, strikethrough, inline code, hyperlinks)
* Lists (ordered, unordered, checkboxes) - to any level of depth
* All headers (header levels >= 3 are treated as header level 3)
* Code blocks
* Block quotes
* Images (where the image is the first element in the paragraph and it has a valid full URL)

## Unsupported Markdown Elements

> *tables*: Tables can be imported in an 'unsupported mode' if you add a flag to the parser.

Reviewer: ```suggestion
*tables*: Tables can be imported in an [unsupported mode](https://developers.notion.com/reference/block) if you add a flag to the parser.
```

This doc link provides some info about the "unsupported" type

First, the default mode - it ignores the tables:

```ts

> const unsupported = false;

Reviewer: ```suggestion
const allowUnsupportedObjectType = false;
```

const blocks: Block[] = markdownToBlocks(`
# Table

| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

> `, unsupported);

Reviewer: ```suggestion
`, allowUnsupportedObjectType);
```

// [
//   {
//     "object": "block",
//     "type": "heading_1",
//     "heading_1": {
//       "text": [
//         {
//           "type": "text",
//           "annotations": {
//             "bold": false,
//             "strikethrough": false,
//             "underline": false,
//             "italic": false,
//             "code": false,
//             "color": "default"
//           },
//           "text": {
//             "content": "Table"
//           }
//         }
//       ]
//     }
//   }
// ]
```

> Next, with unsupported flag = true (note I have removed the `annotations` from the returned object to make it easier to see what is going on):

Reviewer: ```suggestion
Next, with unsupported flag = true (note the `annotations` have been removed from the returned object to make it easier to see what is going on):
```

```ts

> const unsupported = true;

Reviewer: ```suggestion
const allowUnsupportedObjectType = true;
```

const blocks: Block[] = markdownToBlocks(`
# Table

| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

> `, unsupported)

Reviewer: ```suggestion
`, allowUnsupportedObjectType)
```
